server/app/services/applicant/question/EmailQuestion.java

```diff
@@ -46,7 +48,15 @@ public Optional<String> getEmailValue() {
       return emailValue;
     }
 
-    emailValue = applicantQuestion.getApplicantData().readString(getEmailPath());
+    ApplicantData applicantData = applicantQuestion.getApplicantData();
```

Comment:
I really like that we are going from Client data --> application. I am wondering if we should also consider Application --> ClientData. Example- If in the application, they specify the email address as "xx@yy.com" and the Client is missing that info, should we populate it back?

Reply:
That's a great idea! I'll check with product on that.

Reply:
@swatkat1 I tested this out and this was actually mostly how it behaved by default with these changes! I just had to add an update so email is fetched from PAI. I'm still waiting on the final 👍 from product, but it's looking like this is how it will work.